src/main/java/io/ebean/config/JsonConfig.java

```diff
@@ -5,6 +5,9 @@
  */
 public abstract class JsonConfig {
 
+  // According to Intellij, this class has no concrete subclass.
+  // Refactor by extracting the 2 enums?
+
   /**
```

Comment:
As stated by the comment: this class contains 2 Enums that have no extra functionality? Split into 2 separate files?